elmslie-plugin/src/main/resources/templates/PresentationReducer.kt.ftl

```diff
@@ -0,0 +1,37 @@
+package ${presentation_package_name}
+
+import vivid.money.elmslie.core.store.Result
+import vivid.money.elmslie.core.store.StateReducer
+import ${presentation_package_name}.${domain_name}Command.*
+import ${presentation_package_name}.${domain_name}Event.Internal
+import ${presentation_package_name}.${domain_name}Event.Ui
+
+internal object ${domain_name}Reducer : StateReducer<${domain_name}Event, ${domain_name}State,
+        ${domain_name}Effect, ${domain_name}Command> {
+
+    override fun reduce(
+        event: ${domain_name}Event,
+        state: ${domain_name}State
+    ): Result<${domain_name}State, ${domain_name}Effect, ${domain_name}Command> =
+        when (event) {
```

Comment:
without it it would be 
```
override fun reduce(
    event: ${domain_name}Event,
    state: ${domain_name}State
) = when (event) {
    // your code
}
```
Which seems to give more space for the implementation